### PR TITLE
Spectator camera marketplace preparation

### PR DESCRIPTION
**Pull request by zfox23**
I'll revise [the test plan](https://docs.google.com/spreadsheets/d/13BdhQyLyRSQH73d86afnnYe1eEiH3yj9AfC_6I7WViQ/edit#gid=0) to ensure testers of #10875 load the script from this URL:
https://s3-us-west-1.amazonaws.com/hifi-content/zfox/spectator-camera/spectatorCamera.js

The script now also references a new camera model.

### CODE REVIEW

**Comment by zfox23** [comment]
[Marketplace Preparations](https://trello.com/c/rSfUYqhs/249-marketplace-preparations)

File: unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js [hunk 1]
```diff
@@ -344,9 +344,9 @@
     function registerButtonMappings() {
         var VRDevices = Controller.getDeviceNames().toString();
         if (VRDevices) {
-            if (VRDevices.includes("Vive")) {
+            if (VRDevices.indexOf("Vive") != -1) {
```
Comment (thoys) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: `!==`

File: unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js [hunk 2]
```diff
@@ -344,9 +344,9 @@
...
                 controllerType = "Vive";
-            } else if (VRDevices.includes("OculusTouch")) {
+            } else if (VRDevices.indexOf("OculusTouch") != -1) {
```
Comment (thoys) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: Strict? `!==`

Comment (zfox23) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: thank you, you're right!

File: unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js (hunk 1 above)
Comment (zfox23) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: thank you, you're right!

**Comment by zfox23** [comment]
[final version of 3d camera model](https://trello.com/c/rnC3qKBc/235-final-version-of-3d-camera-model)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1646/HighFidelity-Beta-PR10960-5f50f290603beb56b73a5e394b26d438eae250c1.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1646/HighFidelity-Beta-PR10960-5f50f290603beb56b73a5e394b26d438eae250c1.dmg

**Pull request merged**